docs/actions/load-entities-sql.md

```diff
@@ -0,0 +1,45 @@
+---
+id: load-entities-sql
+title: Load Entities (SQL)
+sidebar_label: Load Entities (SQL)
+---
+
+> Audience: [`Citizen Developer`](/docs/audience#citizen-developers)<br/>
+> Skill Prerequisites: `Using Form Builder`
+
+This action creates an entity in the current context from the result of the provided SQL Query. You can find the number of loaded entities in the `[<EntityName>:Count]` token.
+
+:::note
+This creates an entity that can be used ONLY in the current context. This does not create a [Plant an App Entity](/docs/entities).
+:::
+
+## `Typical Use Cases`
+
+- Load values from a SQL Query that returns a list of contacts and then send individual emails to that list.
+- Load values from a SQL Query into an entity and then serialize that entity into CSV or Excel to provide the results in a file to the end-user.
+
+## `Don't use it to`
+
+- Run a SQL query for any other reason than selecting records to create a contextual entity. For other SQL query scenarios, use the [Run SQL Query](/docs/actions/run-sql) action instead.
+
+## `Related Actions`
+
+| Action Name | Description |
+| -- | -- |
+| [Execute Actions on Entity List](/docs/actions/execute-actions-entity-list)   | Execute a list of actions on each item in the specified context entity. Use this to process the items loaded in the Entity created. |
+| [Extend Entity](/docs/actions/extend-entity)   | Extends an entity with new properties. |
+| `Serialize to CSV`   | Serializes a list of entities from the context into CSV (Comma Separated Values) format. |
```

Comment:
Is it safe to assume this one and the following item don't have documentation written yet and that is why they are formatted this way?

Reply:
Yes, that's correct. Perhaps I will get those next! I will go ahead and merge this one.